### ISSUE TITLE
Incorrect StubToServiceMapping for logging

### ISSUE THREAD

**Issue by RedGuys** [yandex-cloud/java-sdk#3]
LogIngestionService and LogReadingService has incorrect mapping.

https://github.com/yandex-cloud/java-sdk/blob/0a9591be7c6e2a4ff3c62cb40bc82e29d1ec2828/java-sdk-services/src/main/java/yandex/cloud/sdk/StubToServiceMapping.java#L244-L246
https://github.com/yandex-cloud/java-sdk/blob/0a9591be7c6e2a4ff3c62cb40bc82e29d1ec2828/java-sdk-services/src/main/java/yandex/cloud/sdk/StubToServiceMapping.java#L247-L249

Needs to be change to
``` java
put(yandex.cloud.api.logging.v1.LogIngestionServiceGrpc.LogIngestionServiceBlockingStub.class, "log-ingestion");
put(yandex.cloud.api.logging.v1.LogIngestionServiceGrpc.LogIngestionServiceFutureStub.class, "log-ingestion");
put(yandex.cloud.api.logging.v1.LogIngestionServiceGrpc.LogIngestionServiceStub.class, "log-ingestion");
put(yandex.cloud.api.logging.v1.LogReadingServiceGrpc.LogReadingServiceBlockingStub.class, "log-reading");
put(yandex.cloud.api.logging.v1.LogReadingServiceGrpc.LogReadingServiceFutureStub.class, "log-reading");
put(yandex.cloud.api.logging.v1.LogReadingServiceGrpc.LogReadingServiceStub.class, "log-reading");
```

https://github.com/yandex-cloud/java-sdk/blob/0a9591be7c6e2a4ff3c62cb40bc82e29d1ec2828/java-sdk-services/src/main/java/yandex/cloud/sdk/ServiceToEndpointMapping.java#L41-L43

For details, refer to the official documentation:
https://cloud.yandex.ru/docs/logging/api-ref/grpc/log_ingestion_service
`A set of methods for writing to log groups. To make a request use ingester.logging.yandexcloud.net.`
https://cloud.yandex.ru/docs/logging/api-ref/grpc/log_reading_service
`A set of methods for reading from log groups. To make a request use reader.logging.yandexcloud.net.`

**Comment by RedGuys** [comment]
Fixed in b6b78338a630dd2b01ea9679ff2d63ce72ec3a00

**Comment by intr13** [comment]
Fixed in 2.5.1 version (com.yandex.cloud:java-sdk)
